src/entity/user/experiences/UserProjectExperience.ts

```diff
@@ -0,0 +1,24 @@
+import { UserExperience } from './UserExperience';
+import { ChildEntity, Column } from 'typeorm';
+import {
+  ExperienceAssociationType,
+  ProjectLink,
+  UserExperienceType,
+} from './types';
+
+@ChildEntity(UserExperienceType.Project)
+export class UserProjectExperience extends UserExperience {
+  @Column({ type: 'jsonb', default: () => [] })
+  links: Array<ProjectLink>;
+
+  // user ids
+  @Column({ type: 'jsonb', default: () => [] })
+  contributors: string[];
+
+  // not adding a relationship, maybe is not important for now
+  @Column({ type: 'jsonb', nullable: true })
+  associatedWith: {
+    type: ExperienceAssociationType;
+    id: string;
+  };
```

Comment:
FK can't be added on jsonb columns I think. Ideally we implement it through referenceId and specific columns workId, and educationId. Is it used anywhere in code, maybe we don't need to define it at all and see later?

Reply:
This is a recurring association field for projects/publications/[...]. 
IMO we should define some kind of relationship, now or during [MI-958](https://dailydotdev.atlassian.net/jira/software/projects/MI/boards/31?selectedIssue=MI-958), because that task will require it.

Maybe we can leave as is and check for it during deletion of experiences using CDC, wdyt?